srcpkgs/singularity/template

```diff
@@ -0,0 +1,58 @@
+# Template file for 'singularity'
+pkgname=singularity
+version=3.7.1
+revision=1
+wrksrc="${pkgname}"
+build_style=go
```

Comment:
This looks like a datapoint for #26417 :p

Reply:
Yes. Similar situation to `aerc`.